.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,14 @@
+name: "CodeQL config"
+
+query-filters:
+  - exclude:
+      problem.severity:
+        - warning
+        - recommendation
+  - exclude:
+      id: go/log-injection
+
+paths-ignore:
+  - bin
+  - '**/*_test.go'
+  - '**/*.test.*'
```

Comment:
I don't see any `*test*` files which aren't `_test.go`. But not a big deal either ways. I'd also suggest to skip `*.txt` files.